### ISSUE TITLE
Expected RUST_DEBUG_HERE_LIFELINE to be defined

### ISSUE THREAD

**Issue by xobs** [ethanpailes/debug-here#14]
In trying to use `debug_here` on Linux, the documentation said to install `debug-here-gdb-wrapper`, then use that to run the program.

The documentation does not say how to run it, so I tried the following:

```
user@Ondo:~/src/my-project$ debug-here-gdb-wrapper cargo run
debug-here-gdb-wrapper: Expected RUST_DEBUG_HERE_LIFELINE to be defined.
               This is a bug with debug-here. Please report it
               at `https://github.com/ethanpailes/debug-here/issues`.





This terminal is just here to display the above error
               message, and won't respond to any input. When you are
               done looking, just close the window.
```

Since that didn't seem to work, I thought perhaps I had to wrap a whole xterm in it. However, that does not work either:

```
user@Ondo:~/src/my-project$ debug-here-gdb-wrapper xterm
debug-here-gdb-wrapper: Expected RUST_DEBUG_HERE_LIFELINE to be defined.
               This is a bug with debug-here. Please report it
               at `https://github.com/ethanpailes/debug-here/issues`.





This terminal is just here to display the above error
               message, and won't respond to any input. When you are
               done looking, just close the window.

```

As such, I am reporting the bug here.

**Comment by ethanpailes** [comment]
Hey @xobs,

You shouldn't need to run the debug-here-gdb-wrapper directly, just put it somewhere on your path. The `debug-here` library itself is the thing that calls the wrapper script. You should use debug here by importing the library and then calling the `debug_here!()` macro.

Also, if you have alacritty installed and on your path, you shouldn't need to install the gdb wrapper at all.

**Comment by xobs** [comment]
I've been using `debug-here` on Windows, and I was trying to figure out how to use it on Linux. I think the README could be updated to say that `debug_here!()` will invoke the command `debug-here-gdb-wrapper` command itself on Linux, and that it's not a wrapper you must manually call.

I originally thought it was a wrapper you ran similar to `rust-gdb`, in that it was a program that set the environment of the program that runs. It seems like the wrapper is actually a way to invoke `gdb` in an xterm and get it to stop at a particular point. It wasn't clear from the README that's what its task was.

I did see the note about alacritty, which is why I tried to use it. However, that gave me a runtime panic, which I documented at https://github.com/alacritty/alacritty/issues/5413
